server/features/features.go

```diff
@@ -0,0 +1,8 @@
+package features
```

Comment:
I think this is a useful addition because there have been a few times where having a feature flag defined in `static.go` hasn't been convenient because of certain dependency chains.

It might be confusing having some flags defined here and some defined in `static.go`, though.

Maybe add a TODO here to move the other feature flags from `static.go` into this file?

Reply:
I added the TODO to the existing flags in `static.go`. Should I submit a follow-up change that moves them?